sklearn/tests/test_multiclass.py

```diff
@@ -48,6 +50,13 @@ def test_ovr_exceptions():
                   np.array([[1.5, 2.4], [3.1, 0.8]]))
 
 
+def test_check_classification_targets():
+    # Test that check_classification_target return correct type. #5782
+    y = np.array([0.0, 1.1, 2.0, 3.0])
+    msg = type_of_target(y)
+    assert_raise_message(ValueError, msg, check_classification_targets, y)
```

Comment:
Can you do something similar to this? (https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/utils/estimator_checks.py#L351)